sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_models.py

```diff
@@ -138,6 +138,13 @@ def _build_item(item):
 class DirectoryProperties(DictMixin):
     """
     :ivar str name: name of the directory
+    :ivar str owner: The owner of the file or directory.
+    :ivar str group: The owning group of the file or directory.
+    :ivar str permissions: Sets POSIX access permissions for the file
+         owner, the file owning group, and others. Each class may be granted
+         read, write, or execute permission.  The sticky bit is also supported.
+         Both symbolic (rwxrw-rw-) and 4-digit octal notation (e.g. 0766) are
+         supported.
```

Comment:
Can we tweak this a little? This looks like the docstring for setting the permissions which doesn't read quite right for a property. Stole this from the REST docs, seems good enough?

> The permissions that are set for user, group, and other on the file or directory. Each individual permission is in [r,w,x,-]{3} format.